src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/model/AdditionalConditionsRequest.kt

```diff
@@ -8,4 +8,8 @@ data class AdditionalConditionsRequest(
   @Schema(description = "The list of additional conditions")
   @field:NotNull
   val additionalConditions: List<AdditionalCondition>,
+
+  @Schema(description = "The type of additional condition, either licence or post sentence supervision", allowableValues = ["AP", "PSS"])
+  @field:NotNull
```

Comment:
Are we expecting there to be different standard and additional conditions by licence type? I don't know the answer to that, but just a thought. We might end up with convoluted config if there is no re-use?

Reply:
My understanding is that there is one list of AP standard/additional conditions, and a list of PSS standard/additional conditions.

AP and PSS licences would have the respective condition types, and AP+PSS would have both